docs/installation.adoc

```diff
@@ -3,13 +3,37 @@
 This document provides information on the installation of `odo` on various operating systems and the location of the binaries and the releases.
 
 * The latest `odo` binaries are available on link:https://dl.bintray.com/odo/odo/latest/[Bintray].
-* The latest `odo` releases are available on
-link:https://github.com/openshift/odo/releases/latest[Github].
-
-== Installing `odo`
-
+* The latest `odo` releases are available on link:https://github.com/openshift/odo/releases/latest[Github].
+
+[[prerequisites]]
+== Prerequisites
+
+OpenShift 3.10.0 or above is required.
+
+[[procedure]]
+== Procedure
```

Comment:
I would skip the title Procedure here, since it is a list of operating systems with a single step in each. Procedure would imply a list of steps to be followed consequentially. Which is why I added Installing `odo`.
How about deleting the sentence on line 15 and adding it as a title: Installation methods for various operating systems
I suggest deleting that line anyway as it repeats information from line 3

Reply:
Ack, makes sense!